### DISCUSSION TITLE
Private docker registry

### DISCUSSION THREAD

**Discussion by npn54** [lfn-cnti/testsuite#1773] [General]
Hi, Im currently using airgapped environment for cnf_testsuites. I'm looking for a way to config a private docker registry for cluster-tools, dockerd, etc...
I found some old issues and pull requests but they're from version 0.2x, like this one [https://github.com/cncf/cnf-testsuite/pull/624/files?diff=split&w=0](url).
The current version's documents also not mentioning about using private docker registry
Is there a way to configure this ? 

Thanks

**Comment by sysarch-repo** [comment]
@namnguyen5497, you can solve this using a mutating webhook (example see https://github.com/Sqooba/k8s-mutate-image-and-policy-webhook). From what I understand, CNF testsuite no longer supports sandbox install.
